website/content/docs/nia/network-drivers/terraform-cloud.mdx

```diff
@@ -0,0 +1,112 @@
+
+Workspace automation requirements for Consul-Terraform-Sync:
+* Must be set to remote execution mode
+* Cannot be connected to a VCS
+* Cannot have an existing configuration version uploaded by another application
+
+Other workspace settings can be pre-configured or updated, such as adding a [run notification](https://www.terraform.io/docs/cloud/workspaces/notifications.html) to send messages to a Slack channel when Consul-Terraform-Sync updates your network infrastructure.
+
+## Configuration Version
+
+An example configuration version for a task named "cts-example" would have the folder structure below when running with the Terraform Cloud driver and using the default working directory.
+
+```shell-session
+$ tree sync-tasks/
+
+sync-tasks/
+└── cts-example/
+    ├── main.tf
+    └── variables.tf
+```
+
+- `main.tf` - The main file contains the terraform block, provider blocks, and a module block calling the module configured for the task.
+  - `terraform` block - The corresponding provider source and versions for the task from the configuration files are placed into this block for the root module.
+  - `provider` blocks - The provider blocks generated in the root module resemble the `terraform_provider` blocks from the configuration for Consul-Terraform-Sync. They have identical arguments present and are set from the intermediate variable created per provider.
+  - `module` block - The module block is where the task's module is called as a [child module](https://www.terraform.io/docs/configuration/modules.html#calling-a-child-module). The child module contains the core logic for automation. Required and optional input variables are passed as arguments to the module.
+- `variables.tf` - This file contains three types of variable declarations: 
+  -  `services` declares input variable that determine module compatibility with Consul-Terraform Sync (read more on [compatible Terraform modules](/docs/nia/terraform-modules) for more details). 
+  - Any additional [optional input variables](/docs/nia/terraform-modules#optional-input-variables) provided by Consul-Terraform-Sync that the module may include. 
+  - Various intermediate variables used to configure providers. Intermediate provider variables are interpolated from the provider blocks and arguments configured in the Consul-Terraform-Sync configuration.
+- `variables.module.tf` - This file is created if there are [variables configured for the task](/docs/nia/configuration#variable_files) and contains the interpolated variable declarations that match the variables from configuration. These are then used to proxy the configured variables to the module through explicit assignment in the module block.
+
+## Variables
+
+Consul-Terraform-Sync uses Terraform input variables to reflect the latest Consul service information and are used as parameters for your Terraform module. These input variables are dynamic and are updated by the driver throughout the runtime of Consul-Terraform-Sync.
```

Comment:
```suggestion
Consul-Terraform-Sync uses Terraform input variables to reflect the latest Consul service information. They are used as parameters for your Terraform module. Input variables are dynamic and are updated by the driver throughout the runtime of Consul-Terraform-Sync.
```
After re-reading, it felt a little clunky to me. Ignore this suggestion if it wasn't bothering you.